calour/io.py

```diff
@@ -393,36 +414,99 @@ def read_amplicon(data_file, sample_metadata_file=None,
 
 
 @ds.with_indent(4)
-def read_open_ms(data_file, sample_metadata_file=None, gnps_file=None, feature_metadata_file=None,
-                 description=None, sparse=False, rows_are_samples=False, mz_rt_sep=None, *, normalize, **kwargs):
-    '''Load an OpenMS metabolomics experiment.
+def read_ms(data_file, sample_metadata_file=None, gnps_file=None, feature_metadata_file=None,
```

Comment:
to be consistent, put `gnps_file` behind `feature_metadata_file`

Reply:
fixed